src/lib/resource/sections/confirmation.js

```diff
@@ -17,10 +13,523 @@ module.exports = {
+                            description: {
+                                adult: `{% set mobilePhoneNumber = "||/answers/p-applicant-confirmation-method/q-applicant-enter-your-telephone-number||" %}
```

Comment:
It would be nice if the content was externalized, folder grouped and replaced with JSON pointer references[ $def and $ref ](https://json-schema.org/understanding-json-schema/structuring.html#id19) perhaps? 
E.g. src/lib/resource/sections/{new folder confirmation}/confirmation-description-[adult|, mainapplicant.incapableAdult....etc].js